packages/website/src/demo-building-blocs/ExampleLayout.tsx

```diff
@@ -33,7 +40,12 @@ export const ExampleLayout: React.FunctionComponent<ExampleLayoutProps> = ({
         <div id={id} className="example-layout">
             <div className="example-layout__header">
                 <h2 className="h4-subdued normal-white-space">{section}</h2>
-                <div>Github</div>
+                <GithubButton
+                    ariaLabel="View source code on GitHub"
```

Comment:
This will make a tooltip no? :eyes: 
Should we use the @plasma/react tooltip? :grin:

Reply:
This is for screen readers, I tried the tooltip but it didn't look good :P

Reply:
hahaha ok :P